src/components/App.js

```diff
@@ -9,61 +9,69 @@ import yaml from 'js-yaml';
 let githubClient;
 
 export default class App extends Component {
-  constructor(props) {
```

Comment:
Interesting, what is the motivation? Years Web used 4 tabs and boom!!! airbnb style changes it to 2 tabs. Lets pray on new standard!!!